setup.py

```diff
@@ -9,7 +9,7 @@
     description="Algorand SDK in Python",
     author="Algorand",
     author_email="pypiservice@algorand.com",
-    version="v1.19.0",
+    version="v1.20.0",
     long_description=long_description,
     long_description_content_type="text/markdown",
```

Comment:
Unrelated to the line - I think `CHANGELOG.md` also requires an update.

@algochoi @jasonpaulos can you align with @algolucky on `CHANGELOG.md` updates?

Reply:
That's correct, @tzaffi made a sub-PR to this PR that added a line in the changelog: 

```
## Changed

- `class StateSchema`'s method `undictify()` now returns a `StateSchema` object instead of a python `dict`
```

We just need to add this in addition to this PR's changelog.

Reply:
how does it look now?